code/API_definitions/CAMARA Connectivity Insights API.yaml

```
@@ -1,46 +1,69 @@
 ---
-    openapi: 3.0.3
+    openapi: 4.0.0
     info:
       title: Connectivity Insights API
       version: '0.3.0-wip'
```

According to new versioning guidelines, work in progress has no version at all.

```suggestion
      version: 'wip'
```